hms-app/src/components/card-details/UserDetails.tsx

```diff
@@ -40,7 +36,6 @@ export default function UserDetails(props: { userId: string }) {
   } as User)
 
   const loadSelectedUser = () => {
-    setIsUserLoading(true)
```

Comment:
why is this deleted?

Reply:
after everytime we edit, there was a little bit of glitch. 
Only after I delete this part, its not happening again

Reply:
could the glitch be the loading time of the user? 
so can we say "while user is loading display loading...."?

Reply:
We already have the that in the return statement. 
I just checked and try to see it in slow motion, and turns out it does shows "User details are loading". But because it only happen for like a few millisecond, we saw it as a glitch.